packages/annotations/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@wordpress/annotations",
+	"version": "1.0.0-beta1",
+	"description": "Annotate content in the Gutenberg editor.",
```

Comment:
This one-liner would have served perfectly fine as a description in the (currently empty-stub) `README.md`

Reply:
> This one-liner would have served perfectly fine as a description in the (currently empty-stub) `README.md`

Fixed in f7d0f1463a83e6589a2ffd15d2b230648dd3c6cc.